<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Google Drive</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="panelsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>90</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsHLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="copyButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F5 Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F6 Move</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newFolderButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F7 New Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F8 Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renameButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F9 Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shareButton">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);
border: 2px solid gray;
border-top-right-radius: 6px;
border-bottom-right-radius: 6px;
border-top-left-radius: 6px;
border-bottom-left-radius: 6px;
width: 16px;</string>
          </property>
          <property name="text">
           <string>F10 Share</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionDownload"/>
    <addaction name="actionUpload"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionMove"/>
    <addaction name="actionNewFolder"/>
    <addaction name="actionDelete"/>
    <addaction name="actionRename"/>
    <addaction name="actionShare"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="separator"/>
   <addaction name="actionDownload"/>
   <addaction name="actionUpload"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.432, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(154, 154, 154, 255));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/settings</normaloff>:/ico/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="iconText">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/download</normaloff>:/ico/download</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/upload</normaloff>:/ico/upload</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/login</normaloff>:/ico/login</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/quit</normaloff>:/ico/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/delete</normaloff>:/ico/delete</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/ico/allitems</normaloff>:/ico/allitems</iconset>
   </property>
   <property name="text">
    <string>Create folder</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="text">
    <string>Share</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
